### PR TITLE
IDEX-2982. Change Che pom.xml parent values

### PR DESCRIPTION
**Pull request by RomanNikitenko**
Change Che pom.xml parent values
Rename module assembly-sdk-war -----> assembly-ide-war
Rename module assembly-sdk -----> assembly-ide

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ok. @vparfonov ?
Check please any problems with this https://github.com/codenvy/che/pull/121

**Comment by vparfonov** [comment]
ok, but author of #121 must update it

**Comment by skabashnyuk** [comment]
PR have to be updated

**Comment by vparfonov** [comment]
ok, wait on merge #121

**Comment by benoitf** [comment]
when this PR will be merged ?

**Comment by RomanNikitenko** [comment]
within a few hours

File: recipes/pom.xml [hunk 1]
```diff
@@ -14,11 +14,11 @@
...
     <parent>
-        <artifactId>che-sdk-parent</artifactId>
-        <groupId>org.eclipse.che.sdk</groupId>
+        <artifactId>che-ide-parent</artifactId>
+        <groupId>org.eclipse.che.ide</groupId>
         <version>3.12.2-SNAPSHOT</version>
     </parent>
     <artifactId>recipes</artifactId>
     <packaging>jar</packaging>
     <name>Che SDK :: Recipes</name>
```
Comment (garagatyi) [inline recipes/pom.xml]: I think `Che SDK` should be changed to `Che IDE` or something like that

File: assembly-sdk/src/assembly/bin/setenv.bat [hunk 2]
```diff
@@ -1,39 +0,0 @@
-@REM
```
Comment (garagatyi) [inline assembly-sdk/src/assembly/bin/setenv.bat]: Looks like you forgot to add that file to index with force parameter

File: assembly-sdk/src/assembly/bin/setenv.sh [hunk 3]
```diff
@@ -1,39 +0,0 @@
-#!/bin/sh
```
Comment (garagatyi) [inline assembly-sdk/src/assembly/bin/setenv.sh]: Looks like you forgot to add that file to index with force parameter

File: assembly-sdk/src/assembly/extInstall.sh [hunk 4]
```diff
@@ -1,43 +0,0 @@
-#!/bin/sh
```
Comment (garagatyi) [inline assembly-sdk/src/assembly/extInstall.sh]: Looks like you forgot to add that file to index with force parameter

**Comment by vparfonov** [comment]
ok

**Pull request merged**